misc/tahoe.spec

```diff
@@ -16,34 +23,45 @@ options = [
     ('W ignore::UserWarning', None, 'OPTION')
 ]
 
-added_files = [
-    ('COPYING.*', '.'),
-    ('CREDITS', '.'),
-    ('relnotes.txt', '.'),
-    ('src/allmydata/web/*.xhtml', 'allmydata/web'),
-    ('src/allmydata/web/static/*', 'allmydata/web/static'),
-    ('src/allmydata/web/static/css/*', 'allmydata/web/static/css'),
-    ('src/allmydata/web/static/img/*.png', 'allmydata/web/static/img')]
+added_files = collect_data_files("allmydata.web")
+
+def collect_dynamic_libs(package):
+    """
+    This is a version of :py:`PyInstaller.utils.hooks.collect_dynamic_libs` that
+    will include linux shared libraries without a `lib` prefix.
+
+    It also only handles dymaic libraries in the root of the package.
+    """
+    base_path, pkg_path = get_package_paths(package)
+    pkg_rel_path = remove_prefix(pkg_path, base_path)
+    dylibs = []
+    for lib_ext in ["*.so", "*.dll", "*.dylib"]:
```

Comment:
As per the last line you removed in `make.bat`, this list should probably contain `*.pyd` also.

Reply:
Done.

Reply:
Thanks!